server/src/main/java/com/google/fhir/gateway/BearerAuthorizationInterceptor.java

```diff
@@ -394,20 +394,16 @@ private void serveWellKnown(ServletRequestDetails request) {
     }
   }
 
-  private void preprocessRequest(
-      ServletRequestDetails servletRequestDetails, AccessDecision accessDecision) {
+  @VisibleForTesting
+  static void mutateRequest(RequestDetails requestDetails, AccessDecision accessDecision) {
```

Comment:
Why is this made `static`? In general please avoid static methods unless if they are really needed, e.g., for simple utility functions. They usually have unit-test complication issues and generally make classes more tightly coupled.